### ISSUE TITLE
FileNotFoundError When Initializing a Lambda Function

### ISSUE THREAD

**Issue by gmolto** [grycap/scar#223]
Attempting to run the ImageMagick example:

```
scar init -s examples/imagemagick/grayify-image.sh -n lambda-imagemagick -es scar-test -i grycap/imagemagick
```
results in the following error:
```
etting udocker environment
Creating function package
Traceback (most recent call last):
  File "/Users/gmolto/Documents/GitHub/grycap/scar/scar.py", line 50, in <module>
    Scar().parse_command_arguments()
  File "/Users/gmolto/Documents/GitHub/grycap/scar/scar.py", line 45, in parse_command_arguments
    self.cloud_provider.parse_command_arguments(args)
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/providers/aws/controller.py", line 93, in parse_command_arguments
    self._lambda.set_properties(args)
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/providers/aws/client/lambdafunction.py", line 353, in set_properties
    self.set_function_code()
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/providers/aws/client/lambdafunction.py", line 265, in set_function_code
    self.properties['code'] = { "ZipFile": utils.get_file_as_byte_array(self.get_property("zip_file_path"))}
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/utils.py", line 89, in get_file_as_byte_array
    with open(file_path, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/var/folders/c_/l6km8t0j78q50zcm7r2h_51m0000gn/T/function.zip'
```

**Comment by robinsingh1** [comment]
I was able to reproduce this issue with this command with the latest master branch

`scar init -n lambda-docker-cowsay -m 128 -t 300 -i grycap/cowsay`

**Comment by gmolto** [comment]
Indeed the problems appears when initializing any function (.e.g `scar init -i ubuntu:16.04` also fails).

Maybe the problem lies on using uDocker locally to generate the container before creating the deployment package. I am running SCAR on macOS and uDocker cannot be run on macOS. (@robinsingh1 which platform are you using?)

If this is so, SCAR should only attempt to use uDocker locally if indicated via a command-line parameter.

**Comment by alpegon** [comment]
Fix in PR #229, the temporal folder path was hardcoded and now is dependent of the OS
